server/src/shared/query_util.js

```diff
@@ -8,6 +8,5 @@ const paramToFilterList = (queryString, filterAllowList) => {
   if (!queryString || queryString === 'undefined') {
     return [];
   }
-
-  return [...new Set(queryString.split(',').filter(param => filterAllowList.has(param)))];
+  return Array.from(filterAllowList).filter(v => queryString.split(',').includes(v));
```

Comment:
IDK why but this was causing issues earlier. maybe something about typescript transpilation.